Jazz-UI automated deployment

Feroz-Shaikh opened the pull request:
### Requirements
npm, node and ng-cli to be installed on the executing system

### Description of the Change
A new Jenkinsfile is proposed to be added for Jazz-UI auto deployment. 

### Benefits
Builds and redeploys Jazz-UI automatically upon a new push to Jazz-UI repo

### Applicable Issues
To implement a production UI quick-fix.

suryajak commented on jazz-ui/Jenkinsfile:
```diff
@@ -0,0 +1,64 @@
+			steps {
+				script {
+					sh "if [ -d jazz-ui ] ; then rm -rf jazz-ui; fi"
+					def var_bitbucket_framework_url = "http://${creds_USR}:${creds_PSW}" + "@" + env.REPO_BASE + "/scm/" + env.REPO_CORE
```
we are trying to **not** name our variables with var_ going forward. Also call it scm_base_url as we now have gitlabs coming in.

Feroz-Shaikh commented:
@suryajak  I have addressed all your comments. 
Had to change the file from declarative pipeline to scripted pipeline due to some limitations in the declarative pipeline.
